src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    @include text-secondary;
+
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    @include text-secondary;
+
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
```

Comment:
The BEM naming convention suggests using a single underscore for element names, like `&-price-text` instead of `&__price-text`. Consider changing this to maintain consistency with BEM.